test/functional/wallet_basic.py

```diff
@@ -496,6 +496,13 @@ def run_test(self):
         self.nodes[0].setlabel(change, 'foobar')
         assert_equal(self.nodes[0].getaddressinfo(change)['ischange'], False)
 
+        # Test "decoded" field value in gettransaction response
+        self.log.info("Testing gettransaction decoding...")
```

Comment:
nit, remove log.

Reply:
Is there a rationale for removing the log ? I added it to mimic the other tests' behavior